util/buildflags/export.go

```diff
@@ -259,9 +260,16 @@ func (w *csvBuilder) Write(key, value string) {
 	if w.sb.Len() > 0 {
 		w.sb.WriteByte(',')
 	}
-	w.sb.WriteString(key)
-	w.sb.WriteByte('=')
-	w.sb.WriteString(value)
+
+	var attr strings.Builder
```

Comment:
Why the extra `strings.Builder` in here? What is the effect of `TrimSpace` in 272?

Reply:
The csv writer creates the proper encoding that handles the commas and the quotes but adds an extra new line at the end so it needs to be trimmed